dockertest/subtest.py

```diff
@@ -291,6 +296,22 @@ def make_subsubtest_config(self, all_configs, parent_config,
         self.config = self.make_config(all_configs, parent_config,
                                        self.config_section)
 
+    def require_docker_version(self, wanted):
```

Comment:
YES! Thank you! That was my first approach, but I backed out after considering that `utils.run("docker")`  might not be the same as `config["docker_path"]`, and DockerVersion has no access to config settings. May I set aside that concern?